src/modules/verifyCardNumber/index.ts

```diff
@@ -1,15 +1,14 @@
 /**
- * Verify Iranian Bank's card number which is valid or not
+ * @description Verify Iranian Bank's card number which is valid or not
  *
  * @public
  * @method verifyCardNumber
- * @param digits
- * @return boolean
+ * @param {Number} digits - card number
```

Comment:
As the JSdoc well-documented all the types(And we have to refactor all modules to make them more human-readable functions in detail), `@params` types should be `lowerCase` except for `Generic types` as well as typescript does.